sirepo/api.py

```diff
@@ -0,0 +1,15 @@
+class APIRequest:
+    def __init__(self):
+        return
+
+    def call_api(self, name, kwargs=None, data=None):
```

Comment:
Write some documentation describing this call. It will be used to call other APIs, checks permissions, etc.

Also add some documentation to uri_router.call_api that it should not be called, and instead self.call_api should be used.

Reply:
3ccfbc6e956420f7da6fc23176cdebb7ae57cfc9